SalesforceCommerceCloud/pwa-kit
# Feature: TAOB & Refresh Token TTL

# Description

This PR implements:

- [x] TAOB in `commerce-sdk-react` (TODO: want feedback from @joeluong-sfcc and others about moving some of the SLAS API path URL as a helper to `commerce-sdk-isomorphic`)
- [x] Configurable Refresh Token TTL in `commerce-sdk-react`

# Test Driving This

1. Go to https://scaffold-pwa-ttl-taob.mobify-storefront.com/
2. Create a user in the way that a customer would create one
3. Create an order and complete the checkout flow
4. Log in via `Trusted Agent` toolbar at the top of the app UI (you will use your Salesforce AM login credentials)
5. Add an item to cart, complete usual checkout flow
6. Go to `Order History` 
7. Verify that you see both the order from the user (not TAOB) and the order from the agent (TAOB)

# Screenshots / Demo


https://github.com/user-attachments/assets/78430402-2cb1-4e92-9306-95a654159c61





# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

- (step1)

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
